build.sbt

```diff
@@ -43,13 +41,8 @@ libraryDependencies ++= Seq(
     exclude("org.typelevel", "cats-laws_2.11")
     exclude("org.typelevel", "cats-kernel-laws_2.11"),
   "ch.qos.logback" % "logback-classic" % "1.2.1" % Provided,
-  "org.webjars" % "swagger-ui" % "2.2.2" % Provided,
-  "io.spray" %% "spray-routing" % sprayV % Provided,
-  "io.spray" %% "spray-http" % sprayV % Provided,
-  "io.spray" %% "spray-can" % sprayV % Provided,
   "com.typesafe.akka" %% "akka-actor" % akkaV % Provided,
-  //---------- Test libraries -------------------//
```

Comment:
You can't figure out that something called "testkit" with "Test" at the end is a test library?